numpy/core/src/common/simd/avx2/arithmetic.h

```diff
@@ -117,3 +117,23 @@
     }
 #endif // !NPY_HAVE_FMA3
 #endif // _NPY_SIMD_AVX2_ARITHMETIC_H
+
+// Horizontal add: Calculates the sum of all vector elements.
```

Comment:
please, move the intrinsics inside the header guard `_NPY_SIMD_AVX2_ARITHMETIC_H`,
same thing for other SIMD extensions.